tests/jq.test

```diff
@@ -2091,3 +2091,18 @@ try ("foobar" | .[1.5]) catch .
 null
 "Cannot index string with number"
 
+# ltrimstr/1 and rtrimstr/1 don't leak, and return an error for
```

Comment:
move to where rest of ltrimstr/rstrimstr tests are?

Reply:
Hmm, there isn't a `ltrimstr`/`rtrimstr` section, only two tests in the middle of "string operations"

```jq
[.[]|ltrimstr("foo")]
["fo", "foo", "barfoo", "foobar", "afoo"]
["fo","","barfoo","bar","afoo"]

[.[]|rtrimstr("foo")]
["fo", "foo", "barfoo", "foobar", "foob"]
["fo","","bar","foobar","foob"]
```

Wouldn't it be confusing if I moved this block with a comment there?

Reply:
That works too i guess, i'm thinking it's good to group tests per function at least? but don't have that strong opinion about it really

Reply:
Hmm, maybe create a separate PR for that, all the string functions are in the same section currently

Reply:
Sounds good 👍